src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -1,16 +1,19 @@
 <template>
-	<www-page class="tw-bg-secondary" style="height: auto;">
-		<div class="tw-max-w-5xl tw-mx-auto tw-p-2 lg:tw-pt-4">
-			<h3 class="tw-text-h3 tw-text-primary">
-				Welcome back, <span class="tw-text-action fs-mask data-hj-suppress">{{ firstName }}</span>
-			</h3>
+	<www-page class="tw-bg-white" style="height: auto;">
```

Comment:
@roger-in-kiva can you use the `main-class` prop here?

Reply:
yeah, done

Reply:
ooh I see is a vue prop in the component and not a class

Reply:
added now

Reply:
Thanks!